tokenserver/tests/assignment/test_sqlnode.py

```diff
@@ -175,10 +175,7 @@ def test_cleanup_of_old_records(self):
         # That should be a total of 7 old records.
         old_records = list(self.backend.get_old_user_records(service, 0))
         self.assertEqual(len(old_records), 7)
-        # And with max_offset of 3, the first record should be id 4
-        old_records = list(self.backend.get_old_user_records(service, 0,
-                                                             100, 3))
-        self.assertEqual(old_records[0][0], 4)
```

Comment:
IIUC it's now safe to restore this test?

Reply:
Ah, indeed! thanks.